medical_center/__manifest__.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Medical Centers",
+    "summary": "Adds a concept of Medical Centers to Patients.",
+    "version": "10.0.1.0.0",
+    "category": "Medical",
+    "website": "https://laslabs.com",
+    "author": "LasLabs, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "depends": [
+        "medical",
+    ],
+    "data": [
+        "views/medical_center.xml",
+        "views/medical_patient.xml",
+        "views/medical_menu.xml",
```

Comment:
@kellylougheed - As PSC lead of this repo, I have overridden the contribution guidelines for file naming that @jcdrubay pointed to. Please switch back to the dedicated menu file.

While the argument can be made that maybe this doesn't make sense for other repos, it most certainly does here. Medical gets deep, and these menus are incredibly difficult to manage. We need the isolation.